### PR TITLE
Enable cleanup and trace overwrite in tufts profile

### PR DESCRIPTION
**Pull request by zhan4429**
---
name: update tufts profile
about: update profile
---

Please follow these steps before submitting your PR:

- [x] If your PR is a work in progress, include `[WIP]` in its title
- [x] Your PR targets the `master` branch
- [x] You've included links to relevant issues, if any

Steps for adding a new config profile:

- [x] Add your custom config file to the `conf/` directory
- [x] Add your documentation file to the `docs/` directory
- [x] Add your custom profile to the `nfcore_custom.config` file in the top-level directory
- [x] Add your custom profile to the `README.md` file in the top-level directory
- [x] Add your profile name to the `profile:` scope in `.github/workflows/main.yml`

### CODE REVIEW

File: conf/tufts.config [hunk 1]
```diff
@@ -33,3 +33,17 @@ singularity {
...
 }
+
+// Perform work directory cleanup when the run has succesfully completed
+trace {
+        trace.overwrite = true
+        enabled         = true
+}
+
+// On a successful completion of a Nextflow run, automatically delete all intermediate files stored in the work/ directory
+cleanup = true
```
Comment (SPPearce) [inline conf/tufts.config:44]: Are you sure you really want this to happen by default?

Comment (jfy133) [reply conf/tufts.config:44]: And it only happens if run completes successfully, which in most cases you assume it should be fine to delte them :+1:

**Review by jfy133 — APPROVED** [review]

**Comment by jfy133** [comment]
@nf-core-bot fix linting

File: conf/tufts.config (hunk 1 above)
Comment (zhan4429) [reply conf/tufts.config:44]: > Are you sure you really want this to happen by default?

Our cluster does not have a purge policy, and the storage is free. Right now limited storage is the No.1 issue. Thanks for advice.

**Comment by zhan4429** [comment]
@jfy133 Hi James, I have an unsuccessful check: Lint tools code formatting / pre-commit (pull_request). Would you guide me on how to fix this? Thanks

**Comment by zhan4429** [comment]
> @jfy133 Hi James, I have an unsuccessful check: Lint tools code formatting / pre-commit (pull_request). Would you guide me on how to fix this? Thanks

I noticed that I have to add a new line otherwise the linting cannot pass. Thanks a lot

**Pull request merged**